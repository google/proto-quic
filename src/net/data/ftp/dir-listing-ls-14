-rwxr-xr-x        0         214      214 Jun 30  2005 !readme
drwxr-xr-x               folder        0 Jul 17  2006 online
-rwxr-xr-x      332           7      339 Feb  5  2004 welcome.txt
