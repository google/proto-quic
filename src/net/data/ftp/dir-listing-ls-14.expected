-
!readme
214
2005
6
30
0
0

d
online
-1
2006
7
17
0
0

-
welcome.txt
339
2004
2
5
0
0
